src/QMCWaveFunctions/Fermion/DiracDeterminant.cpp

```diff
@@ -403,6 +444,8 @@ void DiracDeterminant<DU_TYPE>::mw_evaluateRatios(const RefVector<WaveFunctionCo
     auto& det = static_cast<DiracDeterminant<DU_TYPE>&>(wfc_list[iw].get());
     const VirtualParticleSet& vp(vp_list[iw]);
     const int WorkingIndex = vp.refPtcl - FirstIndex;
+    assert(WorkingIndex >= 0);
+    std::copy_n(det.psiM[WorkingIndex], det.invRow.size(), det.invRow.data());
```

Comment:
macro protect.

Reply:
I'm starting to wonder if this is just a merge issue. I deleted this line earlier.